### PR TITLE
Docs: Lassen HDF5 Perf. Hints

### PR DESCRIPTION
**Pull request by ax3l**
Copy the IBM GPFS Performance hints contributed by @jeanbez that we use on Summit (OLCF) to Lassen (LLNL).

User testing this @joshualudwig8

### CODE REVIEW

**Comment by ax3l** [comment]
@joshualudwig8 please let me know if this improves your performance (after block sizes were adjusted to be ~8 per GPU instead of ~128 per GPU).

**Comment by joshualudwig8** [comment]
@ax3l I saw a significant performance improvement (after block sizes were adjusted to be ~8 per GPU instead of ~128 per GPU). The speedup was almost 2x.
Thanks for the suggestion! 

The HDF5 hints did not have a noticeable impact in speeding up the writing of diag files.

File: Tools/machines/lassen-llnl/lassen.bsub [hunk 1]
```diff
@@ -26,5 +26,26 @@ export OMPI_MCA_io=ompio
 #   https://github.com/ECP-WarpX/WarpX/pull/2874
 export OMPI_MCA_coll_ibm_skip_allgatherv=true
 
+# ROMIO has a hint for GPFS named IBM_largeblock_io which optimizes I/O with operations on large blocks
+export IBM_largeblock_io=true
+
+# MPI-I/O: ROMIO hints for parallel HDF5 performance
+export OMPI_MCA_io=romio321
```
Comment (ax3l) [inline Tools/machines/lassen-llnl/lassen.bsub]: Duplicate (already above)
```suggestion
```

**Comment by ax3l** [comment]
@joshualudwig8 Thanks for testing!

For HDF5 tuning hints, maybe worth opening a ticket with LC to ask for recommendations for _MPI-IO_ tuning on Lassen. Can you take this on and CC me and Jean Luca? https://crd.lbl.gov/divisions/scidata/sdm/staff/jean-luca-bez/

**Comment by ax3l** [comment]
I would merge these options for now nonetheless, since they are good defaults.
We continue digging what can improve your performance beyond using larger blocks (1-8 per GPU).

**Review by RemiLehe — APPROVED** [review]

**Pull request merged**